pkg/job/post_twitter_nft_sales.go

```diff
@@ -0,0 +1,51 @@
+package job
+
+import (
+	"github.com/defipod/mochi/pkg/entities"
+	"github.com/defipod/mochi/pkg/logger"
+)
+
+type postTwitterNftSales struct {
+	entity *entities.Entity
+	log    logger.Logger
+}
+
+func NewPostTwitterNftSales(e *entities.Entity, l logger.Logger) Job {
+	return &postTwitterNftSales{
+		entity: e,
+		log:    l,
+	}
+}
+
+func (c *postTwitterNftSales) Run() error {
+	unnotifiedMessages, err := c.entity.GetAllMochiSalesMessage()
+	if err != nil {
+		c.log.Error(err, "failed to get sales messages")
+		return err
+	}
+	twitterConfigs, err := c.entity.GetAllTwitterConfig()
+	if err != nil {
+		c.log.Error(err, "failed to get twitter configs")
+		return err
+	}
+	for _, t := range twitterConfigs {
+		for _, message := range unnotifiedMessages {
+			err := c.entity.GetSvc().Twitter.SendSalesMessageToTwitter(&message, &t)
+			if err != nil {
+				c.log.Error(err, "failed to get send tweet")
+				return err
+			}
+			err = c.entity.UpdateMochiSalesMessage(message.ID)
```

Comment:
i think after send twitter sales then we delete that
- no need to update `true` and then delete all